napari/_qt/widgets/qt_dims_slider.py

```diff
@@ -70,10 +71,21 @@ def __init__(self, parent: QWidget, axis: int):
         sep.setFixedSize(1, 14)
         sep.setObjectName('slice_label_sep')
 
-        self._fps = 10
+        settings = get_settings()
+
+        # For whatever reason, using settings.application.playback_mode by itself
+        # does not work. But setting the mode this way seems to work.
+        if settings.application.playback_mode == 'loop':
+            loop_mode = LoopMode.LOOP
+        elif settings.application.playback_mode == 'back_and_forth':
+            loop_mode = LoopMode.BACK_AND_FORTH
+        else:
+            loop_mode = LoopMode.ONCE
```

Comment:
yeah, this bugs me too.  the reason is that we have the following config set for our settings (and our evented model).

```py
class Config:
    use_enum_values = True
```

there was a desire earlier on to keep things as plan strings for the outside world... hence the setting... but I think we're reconsidering that.  For now though, you can simplify this as follows (i.e. you can call the enum class with the string value)

```suggestion
        LoopMode(settings.application.playback_mode)
```